Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,6 +40,7 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             LocalIsLatestVersion = info.LocalIsLatestVersion;
             LocalPath = info.LocalPath;
             InstalledDateTime = info.InstalledDateTime;
+            InstallationArgs = info.InstallationArgs;
```

Comment:
Oh, if you're going after that scenario, then it is more problematic. There is no way to update an existing installed extension unless you uninstall it first. At least that's what I recall. Please try that scenario to see if it's a no op or if it automatically does the uninstall/reinstall.